example/extensions/lib_custom_op/README.md

```diff
@@ -0,0 +1,69 @@
+CustomOp Example and Tutorial
+====
+
+## Getting Started
+
+## Have MXNet Ready:
+
+First you should install MXNet either from compiling from source code or download from nightly build. It doesn’t matter if the build comes with CUDA or MKLDNN. The custom operator doesn’t intervene with the execution of other native MXNet operators.
+
+## Run An Example:
+
+You can start getting familiar with custom operator by running some examples we provide in the *example/extensions/lib_custom_op* directory. There are 2 examples: a simple 2D gemm operator, a subgraph operator, and a Makefile.
```

Comment:
lets not mention the subgraph operator here. It will be removed from the custom Op example in the subgraph property PR #17034 and moved to the subgraph property example. It doesnt make sense to have this subgraph op example here since they cannot be used in isolation. Subgraph ops have to be inserted by subgraph properties.